HelperClasses/RequestParameters.cs

```diff
@@ -1,60 +1,68 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Text;
-using System.Threading.Tasks;
 
 namespace EncompassREST.HelperClasses
 {
     public class RequestParameters : List<RequestParameter>
     {
-
         public RequestParameters()
         {
-
         }
 
-        public void Add(string Key, string Value)
+        public void Add(string key, string value)
         {
-            this.Add(new RequestParameter(Key, Value));
+            Add(new RequestParameter(key, value));
         }
 
         public override string ToString()
         {
-            if (this.Count > 0)
+            if (Count > 0)
             {
-                StringBuilder sb = new StringBuilder();
-                sb.Append("?");
-                sb.Append(String.Join("&",this));
+                var sb = new StringBuilder();
```

Comment:
I suppose internally `String.Join` uses a `StringBuilder` so I guess this change wasn't necessary though I don't think you gain anything by using a `StringBuilder` in this case. Just return the string concatenation of the ampersand with the request parameters or use an interpolated string.